### DISCUSSION TITLE
Maybe specify the c++ version using in the project

### DISCUSSION THREAD

**Discussion by chenrui333** [bbuchfink/diamond#661] [Ideas]
👋 I think it might be a good idea to specify the [c++ version used in this project](https://cmake.org/cmake/help/latest/variable/CMAKE_CXX_STANDARD.html). It would also help override the c++ version if needed.

**Comment by bbuchfink** [comment]
I already use the `CMAKE_CXX_STANDARD` call in my cmakelists file.
